Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner
+
+fun main() {
+    // Let we want to find the Prime Numbers below a certain number.
+    val scanner = Scanner(System.`in`)
+    print("Enter an Integer : ")
+    var num = scanner.nextInt()
+    var prime = Array(num){ true }
+    var sqre = square_root(num.toFloat())
+    for (i in 2 until sqre.toInt()) {
+        if (prime[i]) {
+            for (j in (2 * i)..(num) step i) {
+                prime[j] = false
+            }
+        }
+    }
+    // Printing Prime Numbers In the range of (1 to given Number)
+    for (primes in 2 until num) {
+        if (prime[primes]) {
+            print(" $primes")
```

Comment:
Fix spacing around ``$primes`` and add an extra line at the end of file. Rest looks good. :+1:

Reply:
Done